### PR TITLE
Line chart native beta documentation

### PR DESCRIPTION
**Pull request by IIsaku**
Work in progress

### CODE REVIEW

**Review by jordanch — APPROVED** [review]
👍

**Comment by ConnorLand** [comment]
@IIsaku when this can be merged, please let me know and I'll remove WIP and merge.

File: content/refguide/native-styling-refguide.md [hunk 1]
```diff
@@ -1309,6 +1309,87 @@ export myCarouselStyle = {
...
+| `grid` | `paddingBottom` | Applies padding to the bottom side of the grid (number). This is useful to make axis value labels visible. |
+| `grid` | `paddingHorizontal` | Applies padding to the horizontal sides of the grid (number). This is useful to make axis value labels visible. |
+| `grid` | `paddingLeft` | Applies padding to the left side of the grid (number). This is useful to make axis value labels visible. |
+| `grid` | `paddingRight` | Applies padding to the right side of the grid (number). This is useful to make axis value labels visible. |
+| `grid` | `paddingTop` | Applies padding to the top side of the grid (number). This is useful to make axis value labels visible. |
+| `grid` | `paddingVertical` | Applies padding to the vertical sides of the grid (number). This is useful to make axis value labels visible. |
+| `xAxis` | `color` | Applies a color to the axis value labels (string). |
+| `xAxis` | `dashArray` | Applies a pattern of dashes and gaps to the axis line (string containing a [dash pattern](https://www.w3.org/TR/SVG11/painting.html#StrokeDasharrayProperty)). |
+| `xAxis` | `fontFamily` | Applies a font types to the axis value labels (string). |
```
Comment (jordanch) [inline content/refguide/native-styling-refguide.md:1364]: types -> type

**Review by jordanch — CHANGES_REQUESTED** [review]
typo

File: content/refguide/native-styling-refguide.md (hunk 1 above)
Comment (IIsaku) [reply content/refguide/native-styling-refguide.md]: I have changed the description a bit.

**Comment by jordanch** [comment]
👍

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**